### ISSUE TITLE
[Feature Request] Include already Lutris installed GOG games

### ISSUE THREAD

**Issue by LinuxfarmerHH** [sharkwouter/minigalaxy#591]
Linuxmint 21.3
Minigalaxy 1.2.5

Have installed 5 games including big whoppers like BG3 with Lutris. So i set the intall path in Minigalaxy to the same top folder, but these installed are not found. I can only push the "install" button again and the whole games will be downloaded a second time.

Searched for some kind of config to point to the installed folder, what does not exist now.

Is there a workaround or idea, how to include the Lutris installed GOG Games into Minigalaxy?

**Comment by sharkwouter** [comment]
It depends. If the game is installed in the install path of Minigalaxy, it might be detected just fine. Otherwise, you can perhaps add a `gameinfo` file to the game directories that looks something like this:
```
Blackwell Convergence
2.3
65124
```
The top one is the name of the game, second one the version and the last the game id. You can find all this info at: https://www.gogdb.org/

Hope that helps. At the moment I don't know another workaround. I guess renaming the directory to match the name on GOG would also work in most cases.

**Comment by GB609** [comment]
That won't be enough. 
The wineprefix folder is not identical to the game folder for minigalaxy.
We're using `<gamefolder>/prefix`. So, unless you can point lutris to the some prefix directory, minigalaxy and lutris will use different prefixes and thus different configurations.
If it is a game that also needs registry keys, it'll very likely not work if you don't point both applications to the same prefix directory.

**Comment by sharkwouter** [comment]
Yeah, I'm going to close this one. This is not a feature that many people will use I think and it is really difficult to implement.
